legion/engine/physics/colliders/convexcollider.hpp

```diff
@@ -15,6 +15,8 @@ namespace legion::physics
     {
     public:
 
+        std::vector<math::vec3> vertices;
```

Comment:
This can perhaps be moved to private again, though it would not be too weird that the vertices can be accessed